### DISCUSSION TITLE
Przewidywanie dostępności źródeł

### DISCUSSION THREAD

**Discussion by tomaszg61112** [TowarzyszFatCat/doccli#11] [Ideas]
Program bardzo mi się podoba, niestety jest jedno ale. Program crashuje się po wejściu w źródło które istnieje tylko z nazwy (np. jest link ale materiał został już z niego usunięty). Fajnie byłoby gdyby doccli przewidywało że takie źródło nie zadziała i wypisało to gdzieś.

**Comment by TowarzyszFatCat** [comment]
Dzięki. Mam takie coś w planach, ale wymaga to odemnie przebudowy całej aplikacji, bo na ten moment kod jest dosyć kiepski pod nowe "ficzery". Ale niedługo chcę się zabrać za przepisanie programu i dodaniu kilku podobnych opcji
